apps/server/scripts/seed/seedParticipant.ts

```diff
@@ -34,6 +34,10 @@ const seedParticipant = async () => {
   const participant = await createParticipant(payload1);
   const participant2 = await createParticipant(payload2);
 
+  event.participants = [participant._id, participant2._id];
+
+  await event.save();
+
```

Comment:
prefer to use findOneAndUpdate and $push https://www.mongodb.com/docs/manual/reference/operator/update/push/

Reply:
sure, I can use findOneAndUpdate and $push instead